src/XML/src/FMI3/fmi3_xml_parser.c

```diff
@@ -238,6 +238,13 @@ const fmi3_xml_primitive_types_t PRIMITIVE_TYPES = {
         0, /* N/A */
         0,
         fmi3_base_type_bool,
+    },
+    {
+        "Enumeration",
+        sizeof(fmi3_int32_t),
+        fmi3_bitness_32, /* TODO: Perhaps this should be 64bit in FMI3??? */
```

Comment:
Note that the FMI standard says:
`To set or inquire variables of type [Enumeration](https://fmi-standard.org/docs/3.0/#figure-schema-Enumeration), fmi3SetInt64 and fmi3GetInt64 must be used.`

Reply:
Fixed in #6ba08da